### PR TITLE
feat: Added prop-types devDependency  in package.json

### LINKED ISSUE #304: [BUG]: `prop-types` devDependency not in package.json
### Description

The package `prop-types`, which is used for prop validation is not added to package.json, and when running in the dev mode (e.g running `pnpm start`) throws an error:

```bash
Error: The following dependencies are imported but could not be resolved:

  prop-types (imported by /.../whisper/client/src/components/MessageStatus.jsx)
```

### Screenshots

_No response_

### Additional information

To fix this, just run `npm i -D prop-types` in the client folder
```
npm i -D prop-types
```

### 🥦 Browser

Other

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

### PR DESCRIPTION
**Pull request by Chetax**
# Fixes  #304 
# 👨‍💻 Changes proposed(What did you do ?)
 Added  prop-types devDependency , Which Is not in package.json 

# ✔️ Check List (Check all the applicable boxes)

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.

### CODE REVIEW

**Comment by Dun-sin** [comment]
Your PR is wrong, read the guidelines [👉🏽here👈🏽](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md#rules) your PR wouldn't be merged if you don't

File: server/.env_sample [hunk 1]
```diff
@@ -1,4 +1,4 @@
 THIS IS JUST A FILE TO HELP YOU KNOW WHAT IS NEEDED IN THE .env FILE.
```
Comment (Dun-sin) [inline server/.env_sample:1]: Remove this file, it's not part of your PR

Comment (Chetax) [reply server/.env_sample]: Mam Can You Please Review One More Time Please

**Comment by Dun-sin** [comment]
@Chetax You can't do it like this, it will delete the file in the main project and we don't want that. You have to find another way try researching how on Google or chatgpt

**Comment by Chetax** [comment]
@Dun-sin  ,creating new pr and closing this one .Thanks !

**Comment by Chetax** [comment]
@Dun-sin ,creating new pr and closing this one .Thanks !

**Pull request closed without merging**